### ISSUE TITLE
Update plugin to latest matter-js version

### ISSUE THREAD

**Issue by akehir** [liabru/matter-attractors#11]
main.js:5301 matter-js: Plugin.use: matter-attractors@0.1.4 is for matter-js@^0.12.0 but installed on matter-js@0.14.2.

**Comment by philipaarseth** [comment]
+1

**Comment by treardon17** [comment]
@liabru is this project still alive?

**Comment by liabru** [comment]
@treardon17 Yes this plugin is supported on the latest version and will continue to work fine! 

I just need to get around to bumping the version range to avoid this warning (which recently I implemented a solution to allow that).

**Comment by treardon17** [comment]
@liabru awesome! This is such a cool library/package by the way. Thanks for contributing it to the community!

**Comment by ggorlen** [comment]
When the version update occurs, we should remember to modernize the [examples](https://github.com/liabru/matter-attractors/blob/3fc7aa5fecde1d1351f7662b21f3977e586fbeb3/docs/examples/basic.js) to avoid [deprecated `World` and `engine.world;`](https://brm.io/matter-js/docs/classes/World.html).
